### PR TITLE
JS Week2 Homework

### PR DESCRIPTION
**Pull request by negaren**

### CODE REVIEW

**Comment by negaren** [comment]
JS1 Week2 Homework

File: javascript/javascript1/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+//Flight booking fullname function - Second method
+function getFullname2(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return "Lord/Lady " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+
+let fullname1 = getFullname2("Negar", "Namini", true);
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:22]: you're not changing fullname1 value, so change it to const. always use const first, until it doesn't let you.

File: javascript/javascript1/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+let fullname2 = getFullname2("Negar", "Namini");
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:23]: const

File: javascript/javascript1/week2/index.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+console.log(fullname1);
+console.log(fullname2);
+
+//.Event application------------------------------------------------------------------------
+let today = new Date();
+const weekDays = ["Monday", "Tuesday", "Wednesday", "Tursday", "Friday", "Saturday", "Sunday"]
+function getEventWeekday(dayNumber) {
+    const toDayNumber = today.getDay();
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:31]: everywhere you are using toDayNumber with decrease one. so you can do it here l mean
const toDayNumber = today.getDay() -1;

File: javascript/javascript1/week2/index.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    if (dayNumber == 7) {
+        console.log("Today is " + '"', weekDays[toDayNumber - 1], '"' + " and the event day is a week later the same day!");
+    } else if (dayNumber < 7) {
+        console.log("Today is " + '"', weekDays[toDayNumber - 1], '"' + " and the event day is on:" + '"', weekDays[toDayNumber + dayNumber - 1], '"');
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:35]: this solution doesn't work properly if the value of `toDayNumber + dayNumber - 1` be more than 6 (size of array). fx  toDayNumber = 6, dayNumber= 2

File: javascript/javascript1/week2/index.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    } else {
+        console.log("Today is " + '"', weekDays[toDayNumber - 1], '"' + " and the event day is on: " + '"', weekDays[(dayNumber % 7) + toDayNumber - 1], '"');
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:37]: same toDayNumber = 6, dayNumber= 9

you can find a right index like this (dayNumber+toDayNumber)%7

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :+1:

**Pull request merged**